src/lib/SearchMap.js

```diff
@@ -250,7 +251,19 @@ export default class SearchMap {
      *
      */
     globalCoordinatesFromExtent(west, south, east, north) {
-        return [[west, south], [east, south], [east, north], [west, north], [west, south]];
+        return [[[west, south], [east, south], [east, north], [west, north], [west, south]]];
```

Comment:
So this now needs an array of arrays of arrays? Wow!

Reply:
Ah, I think I see the connection to the changes in function calls now.

Reply:
Yeah, before the "outer" array was wrapped by the caller, not sure why.  Polygons have an array of array of coordinates - the first of the "outer" array is the polygon itself, and subsequent entries are "holes" in the polygon.  We aren't using holes so we just don't provide the other entries.

MultiPolygons have one more level, as each is a separate polygon.